### ISSUE TITLE
RetrieveUserPrivileges

### ISSUE THREAD

**Issue by scottdurow** [kenakamu/CRMLinqPadDriverWebAPI#12]
when I execute the following:
var u = (from a in systemusers where a.lastname=="Durow" select a).First();
var privs = u. RetrieveUserPrivileges().GetValue();
privs.Dump();

I get the error;
Resource not found for the segment 'Microsoft.Dynamics.CRM.RetrieveUserPrivileges'

It looks like it's because it's executing the function as an unbound function rather than a bound function.

Am I doing something wrong?

**Comment by kenakamu** [comment]
Sorry for late reply. Try this.

var u = (from a in systemusers where a.lastname=="Durow" select a).First();
var privs = systemusers.ByKey(u.systemuserid). RetrieveUserPrivileges().GetValue();
privs.Dump();

**Comment by DineshNanayakkara** [comment]
Hi,

I got the following error even I tried your code sample.

Encountered the following errors when parsing the EDMX document: UnexpectedXmlElement : The schema element 'NavigationProperty' was not expected in the given context. : (1, 2083309)UnexpectedXmlElement : The schema element 'NavigationProperty' was not expected in the given context. : (1, 2228794)UnexpectedXmlElement : The schema element 'NavigationProperty' was not expected in the given context. : (1, 2249340) | Encountered the following errors when parsing the EDMX document: UnexpectedXmlElement : The schema element 'NavigationProperty' was not expected in the given context. : (1, 2083309)UnexpectedXmlElement : The schema element 'NavigationProperty' was not expected in the given context. : (1, 2228794)UnexpectedXmlElement : The schema element 'NavigationProperty' was not expected in the given context. : (1, 2249340)
-- | --
Encountered the following errors when parsing the EDMX document: UnexpectedXmlElement : The schema element 'NavigationProperty' was not expected in the given context. : (1, 2083309)UnexpectedXmlElement : The schema element 'NavigationProperty' was not expected in the given context. : (1, 2228794)UnexpectedXmlElement : The schema element 'NavigationProperty' was not expected in the given context. : (1, 2249340)
